.github/workflows/GitHub_Action.yml

```diff
@@ -0,0 +1,25 @@
+name: GitHub_Action
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  GitHub_Action:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]  # , pypy3]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+      - run: pip install codespell flake8 isort mypy pytest
+      - if: matrix.python-version >= 3.6
+        run: |
+          pip install black
+          black --check .
+      - run: codespell --ignore-words-list="bloaded,nto,pullrequest,pullrequests,thi,tim,wan" --quiet-level=2 --skip="./tests/*" || true
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: isort --recursive .
+      - run: pip install httpretty -r requirements.txt
```

Comment:
I think the test failures are due to not installing the test requirements. httpretty and pytest is included in them, so you can drop it from the above steps:

``pip install -r requirements.txt -r test-requirements.txt``